docs/users/Dependency_Versions.mdx

```diff
@@ -7,7 +7,7 @@ import packageJson from '../../package.json';
 
 ## ESLint
 
-> The version range of ESLint currently supported is `^6.0.0 || ^7.0.0 || ^8.0.0`.
+> The version range of ESLint currently supported is `^7.0.0 || ^8.0.0`.
```

Comment:
i went to update this file cos I'd forgotten we'd automated it via importing the `package.json`. But I noticed this was wrong so I fixed it.